python/py-llvmlite-devel/Portfile

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        numba llvmlite 409cd6d37fe1ba76438aa1a3a9c6a8adcd0ddb9d
+fetch.type          git
+name                py-llvmlite-devel
+conflicts           py310-llvmlite
+set my_name         py310-llvmlite
```

Comment:
If the software specifies a development/prerelease version, then I think it would be nice for the port version to at least begin with it (0.56 in this case); I would think that makes it easier to compare against the non-devel port.

Reply:
@chrstphrchvz I am not sure I understand this comment - where does the 0.56 number come from? Also I don't think we should use a version number for pre-release versions (afaik we don't do that anywhere else either) since we have no way of telling what the version number is upstream will decide to release as the next stable version. 
Or am I misunderstanding here what you're suggesting?

Reply:
The PR description mentioned 0.56, but more precisely I meant whichever version the software reports once installed, i.e. whatever is in `numba.__version__`.

There are some devel ports which do contain a software-reported version in the `version` field, and others which don’t. My impression is it is often arbitrary or decided case-by-case; if there should be a consistent policy (if there isn’t already one which I wasn’t aware of), it would probably be better to discuss it on the mailing list.

Reply:
my personal opinion is not to add a version number for -devel ports, but am not aware if a policy (nor do I think we need one). But feel free to bring this up on the mailinglist if it's important to you.